src/admin/main.cc

```diff
@@ -80,6 +81,7 @@ int main(int argc, const char** argv) {
 		if (argc < 2) {
 			throw WrongUsageException("No command name provided");
 		}
+		socketinit();
```

Comment:
I can see `socketinit` function implementation is needed on Windows platform only. Please review if as a result you also need to call for function `socketrelease` as well on Windows.

Reply:
Done